docs/GettingStarted.md

```diff
@@ -9,12 +11,12 @@ If you want to get started quickly and easily then we recommend just using Vagra
 If you have difficulty, try the [Troubleshooting Guide](./Troubleshooting.md).
 
 ### A note about names & addresses
-In this example, we will use the server names and IP addresses from the [Calico CoreOS Vagrant][calico-coreos-vagrant] example.
+In this example, we will use the server names and IP addresses from the [Calico Ubuntu Vagrant][calico-ubuntu-vagrant] example.
 
 | hostname | IP address   |
 |----------|--------------|
-| core-01  | 172.17.8.101 |
-| core-02  | 172.17.8.102 |
+| ubuntu-0 | 172.17.8.100 |
```

Comment:
Why are we using different addressing between coreos and ubuntu. THat just seems silly.